sequencer/src/persistence.rs

```diff
@@ -176,16 +216,19 @@ mod persistence_tests {
             recipient_key: pubkey,
         };
         let mut quorum_proposal = Proposal {
-            data: QuorumProposal::<SeqTypes> {
+            data: QuorumProposal2::<SeqTypes> {
                 block_header: leaf.block_header().clone(),
                 view_number: ViewNumber::genesis(),
                 justify_qc: QuorumCertificate::genesis::<TestVersions>(
                     &ValidatedState::default(),
                     &NodeState::mock(),
                 )
-                .await,
+                .await
+                .to_qc2(),
                 upgrade_certificate: None,
-                proposal_certificate: None,
+                view_change_evidence: None,
+                drb_seed: [0; 32],
```

Comment:
can we use `hotshot_types::drb::INITIAL_DRB_SEED_INPUT` and `hotshot_types::drb::INITIAL_DRB_RESULT` for the constants?

they should be the same thing but we may still change the "default" type/number